haiku-apps/genio/genio-3.0.0.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="The Haiku IDE"
+DESCRIPTION="Genio is a native and fully functional IDE for the Haiku.
+Some of the features of the Genio IDE are:
+*   LSP Server support (autocompletion, signature help, go to definition/implementation/declaration, quick fix, format, rename)
+*   Multi-project browser
+*	Class outline
+*   Integrated source control with GIT (including opening a remote project)
+*   Links to file and build errors in Build Log and Console I/O
+*   Build on save / Save on build
+*   User templates for quickly creating new files and projects
+*   Rich editor with many features, like syntax highlighting
+*   Full screen and Focus mode"
+HOMEPAGE="https://github.com/Genio-The-Haiku-IDE/Genio/releases"
+COPYRIGHT="2022-2024 The Genio Team"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Genio-The-Haiku-IDE/Genio/archive/v3.0.tar.gz"
+CHECKSUM_SHA256="64cb18b552a6336e22903b6fc659253bf1ad02831fb2bc4dd892e7e65023465c"
+SOURCE_FILENAME="Genio-v3.0.tar.gz"
+SOURCE_DIR="Genio-3.0"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	genio${secondaryArchSuffix} = $portVersion
+	app:Genio = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	gcc${secondaryArchSuffix}_syslibs_devel
+	cmd:clang >= 16
+	lib:libeditorconfig$secondaryArchSuffix
+	lib:liblexilla$secondaryArchSuffix >= 5.2.4
```

Comment:
>= 5.2.4 shouldn't be needed.

Reply:
you mean I should write "> 5.2.4" or remove the check altogether ?